Romuashi/hometask-3/rainfall.rb

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'colorize'
+
+def data
+  file = File.new('data')
+  file.read
+end
+
+def mean(town, data)
+  return -1 unless data.include?(town)
+
+  search_town(town, data).sum / 12
+end
+
+def variance(town, data)
+  return -1 unless data.include?(town)
+
+  search_town(town, data).map { |city| (mean(town, data) - city)**2 }.sum / 12
+end
+
+def search_town(town, data)
+  data.split("\n").each do |line|
+    return line.scan(/\d*\.\d/).map(&:to_f) if line.include?(town)
+  end
+end
+
+def run_cli
+  loop do
+    p 'Enter your string: '
```

Comment:
it's a good practice to create constants and use them instead of string or integer literals. Apply it everywhere in the code where it possible.
example:
```
ENTER_STRING = 'Enter your string: '
p ENTER_STRING